sway/handlers.c

```diff
@@ -220,6 +220,10 @@ static bool handle_view_created(wlc_handle handle) {
 			// refocus in-between command lists
 			set_focused_container(newview);
```

Comment:
Probably a wlc issue tbh. It draws them before we can arrange them. I keep asking @Cloudef about this but it's difficult to fix.